scripts/scil_perform_majority_vote.py

```diff
@@ -0,0 +1,126 @@
+    p.add_argument('--output_prefix', default='voting_',
+                   help='Output prefix, [%(default)s].')
+
+    add_overwrite_arg(p)
+
+    return p
+
+
+def main():
+    parser = _build_args_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(parser, args.bundles)
+    output_streamlines_filename = '{}streamlines.trk'.format(
+        args.output_prefix)
+    output_voxels_filename = '{}voxels.nii.gz'.format(args.output_prefix)
+    assert_outputs_exist(parser, args, [output_voxels_filename,
+                                        output_streamlines_filename])
+
+    if not 0 <= args.ratio_voxels <= 1 or not 0 <= args.ratio_streamlines <= 1:
+        parser.error('Ratios must be between 0 and 1.')
+
+    fusion_streamlines = []
+    for name in args.bundles:
+        fusion_streamlines.extend(
+            load_tractogram_with_reference(parser, args, name).streamlines)
+
+    fusion_streamlines, _ = perform_streamlines_operation(union,
+                                                          [fusion_streamlines],
+                                                          0)
+    fusion_streamlines = ArraySequence(fusion_streamlines)
+    if args.reference:
+        reference_file = args.reference
+    else:
+        reference_file = args.bundles[0]
+
+    transformation, dimensions, _, _ = get_reference_info(reference_file)
+    volume = np.zeros(dimensions)
+    streamlines_vote = dok_matrix((len(fusion_streamlines),
+                                   len(args.bundles)))
+
+    for i, name in enumerate(args.bundles):
+        if not is_header_compatible(reference_file, name):
+            raise ValueError('Both headers are not the same')
+        sft = load_tractogram_with_reference(parser, args, name)
+        bundle = sft.get_streamlines_copy()
+        sft.to_vox()
+        bundle_vox_space = sft.get_streamlines_copy()
+        binary = compute_robust_tract_counts_map(bundle_vox_space, dimensions)
+        volume[binary > 0] += 1
+
+        if args.same_tractogram:
+            _, indices = perform_streamlines_operation(intersection,
+                                                       [fusion_streamlines,
+                                                        bundle], 0)
+            streamlines_vote[list(indices), i] += 1
+
+    if args.same_tractogram:
+        real_indices = []
+        for i in range(len(fusion_streamlines)):
+            ratio_value = int(args.ratio_streamlines*len(args.bundles))
+            if np.sum(streamlines_vote[i]) >= ratio_value:
+                real_indices.append(i)
+
+        new_streamlines = fusion_streamlines[real_indices]
+
+        sft = StatefulTractogram(new_streamlines, reference_file, Space.RASMM)
+        save_tractogram(sft, output_streamlines_filename)
+
+    volume[volume < int(args.ratio_streamlines*len(args.bundles))] = 0
+    volume[volume >= int(args.ratio_streamlines*len(args.bundles))] = 1
```

Comment:
Here, why not simply do
`volume[volume > 0] = 1`
? Since everything that was already under the ratio was set to 0 in the previous call.